BlargBucket/ViewControllers/ReadMeViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ReadMeViewController.swift
+//  BlargBucket
+//
+//  Created by Nick Smillie on 1/1/15.
+//  Copyright (c) 2015 Luke Deniston. All rights reserved.
+//
+
+import UIKit
+
+class ReadMeViewController: UIViewController {
+	
+	@IBOutlet var textView: UITextView!
+	
+	var viewModel = ReadMeViewModel()
+	
+	/**
+	Override initializer to set the nib and add an observer for fetching the readme
+	*/
+	override init() {
+		super.init(nibName: "ReadMeView", bundle: nil)
+		viewModel.addObserver(self, forKeyPath: "readMeMarkDown", options: .New, context: nil)
```

Comment:
Make `"readMeMarkDown"` a private property. This prevents lame errors where somewhere we use "readmeMarkDown" instead and then we spend five hours trying to figure out why things aren't working.